### ISSUE TITLE
Any plan to add RPM packages for RedHat/CentOS

### ISSUE THREAD

**Issue by thru67** [certtools/intelmq#2335]
Hello,

I see that the RPM for RedHat/CentOS is missing in the stable repo. Is there any plan to support RPM for RedHat/CentOS in the future?

/Regards

**Comment by sebix** [comment]
Hi @thru67 

I provided packages for CentOS 7 / RHEL 7 in the repositories in the past, but the version 7 became too outdated for IntelMQ's requirements.
CentOS 8 / Stream requires different packaging approaches, and Rocky/Alma Linux are again a bit different.
When you look at the [spec file](https://build.opensuse.org/package/view_file/home:sebix:intelmq/intelmq/intelmq.spec?expand=1) you can see that there are still leftovers from that time.
The [unstable repository](https://build.opensuse.org/project/show/home:sebix:intelmq:unstable) does have CentOS 8, but it fails to build because of unresolved dependencies. That shows that most effort must be put into the dependencies rather than packaging IntelMQ itself.

To sum it up: It needs a bit of effort but can be done, proven by the fact that we had it in the past.

If you can contribute to the packaging of IntelMQ, let me know.

**Comment by sebix** [comment]
Closing because of inactivity. Please reopen if needed.
